src/Internal/DependencyManagement/CoreServiceProvider.php

```diff
@@ -250,6 +256,7 @@ function ( ...$arguments ) {
 		$this->share_with_tags( ReviewAfterConversionsNote::class, MerchantMetrics::class, WP::class );
 		$this->share_with_tags( SetupCampaignNote::class );
 		$this->share_with_tags( SetupCampaign2Note::class );
+		$this->share_with_tags( SetupCouponSharing::class );
```

Comment:
Preferable to use `SetupCouponSharingNote`
```suggestion
		$this->share_with_tags( SetupCouponSharingNote::class );
```